catalyst/contrib/nn/criterion/triplet.py

```diff
@@ -1,10 +1,15 @@
 # flake8: noqa
```

Comment:
could you please remove `flake8: noqa`?

Reply:
this `noqa` is needed for another code in the module, not for mine